core/server/web/shared/middlewares/image/handle-image-sizes.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const sharp = require('sharp');
+const storage = require('../../../../adapters/storage');
+const activeTheme = require('../../../../services/themes/active');
+
+const SIZE_PATH_REGEX = /^\/size\/([^/]+)\//;
+
+module.exports = function (req, res, next) {
+    if (!SIZE_PATH_REGEX.test(req.url)) {
+        return next();
+    }
+
+    const imageSizes = activeTheme.get().config('image_sizes');
+    const imageDimensions = Object.keys(imageSizes).reduce((dimensions, size) => {
```

Comment:
This statement is failing when `image_sizes` is not found in the config (can't do Object.keys(undefined)). We probably need a default for `image_sizes` config or skip processing altogether when it's not defined?

Reply:
Nice catch - that should skip processing :+1: